bidwire/scrapers/massgov/url_scraper_dict.py

```diff
@@ -0,0 +1,42 @@
+UL_CATEGORY_LI = '//ul[@class="category"]/li'
```

Comment:
I'm not sure this needs to be its own file, it could be in the `..._scraper.py` file (but I'm fine with it either way).